lib/validator/sfValidatorDate.class.php

```diff
@@ -156,7 +156,7 @@ protected function convertDateArrayToString($value)
         // all elements must be empty or a number
         foreach (array('year', 'month', 'day', 'hour', 'minute', 'second') as $key) {
             if (isset($value[$key]) && !ctype_digit((string) $value[$key]) && !empty($value[$key])) {
-                throw new sfValidatorError($this, 'invalid', array('value' => $value));
+                throw new sfValidatorError($this, 'invalid', $value);
```

Comment:
This will break `sfValidatorError::getValue()` that depends on `value` key.

Reply:
unfortunately that's not covered by the tests, as they're green anyway :(

Reply:
That's human mutation testing. :)